packages/lab/src/nav-item/NavItem.css

```diff
@@ -0,0 +1,81 @@
+.saltNavItem {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  background: var(--salt-navigable-primary-background);
+  gap: var(--salt-spacing-75);
+}
+
+.saltNavItem-horizontal {
+  min-height: calc(var(--salt-size-stackable) + var(--salt-size-accent) + var(--salt-spacing-100));
```

Comment:
```suggestion
  min-height: calc(var(--salt-size-stackable) + var(--salt-size-bar) + var(--salt-spacing-100));
```

Reply:
new token update + stackable will be deprecated, i think it should be size-base and then padding above and below of spacing-50, or whatever this is calculated to be

Reply:
Not sure if it's worth tackling this atm, versus when we have design guidance

Reply:
np